src/rtr/pdu.rs

```diff
@@ -1007,6 +1020,18 @@ impl ProviderAsns {
     }
 }
 
+#[cfg(feature = "arbitrary")]
+impl<'a> arbitrary::Arbitrary<'a> for ProviderAsns {
+    fn arbitrary(
+        u: &mut arbitrary::Unstructured<'a>
+    ) -> arbitrary::Result<Self> {
+        let size = usize::arbitrary(u)? % (
+            usize::from(u16::MAX) * mem::size_of::<Asn>()
+        );
```

Comment:
This does not guarantee that `size` is a multiple of 4 (i.e. `mem::size_of::<Asn>()`), won't this result in invalid `ProviderAsns`?

Reply:
Just move the parentheses, right?